config/locales/voc.en.yml

```
@@ -10,6 +10,7 @@ en:
     follow: "Follow"
     load: "Load more"
     login: "Sign in"
+    logout: "Logout"
```

```suggestion
    logout: "Sign out"
```

To match "Sign in"/"Sign up"